digdag-docs/src/operators/bq_load.md

```diff
@@ -232,37 +232,37 @@
 
   You can write schema within .dag file directly.
 
-  ```yml
+  ```yaml
   +step:
     bq_load>: gs://<bucket>/path/to_file
     ...
     schema:
       fields:
-        - name: "name",
+        - name: "name"
           type: "string"
         ...
   ```
 
   Or you can write it as external file.
 
-  ```json
+  ```java
```

Comment:
As you might know, it seems like `Could not lex literal_block as "json"` is kind of syntax error because the content cannot be parsed by JSON parser. Included `,` or `...` like that.
ref: https://github.com/sphinx-doc/sphinx/issues/2413

We could keep `json` type if we would fix the JSON content. But, the current content also looks good to me. So, you think that we'd better to change type with `java` to make highlighting?

Reply:
Got it. I did compare highlights between `java` and `json` for this JSON content. I think that we'd better to use java highlighting and `...`. I will put LGTM on this PR. 

![](https://i.gyazo.com/df462721c3626f9255d2a833cd8c7f68.png)
^ java highlighting

![](https://i.gyazo.com/db23a1a5ae6e0a9a672d395335f964af.png)
^ json highlighting

Reply:
> I'm not sure why but current digdag.io documents doesn't highlight codes. (It's configuration problem?)

Could you show me more concretely? I still getting your above question.

Reply:
@muga Thank you for the comment. I misunderstood.

I thought digdag document exist code identifier like the below. And It's not highlighted.

    ```json

But those are no code identifier except this PR.

    ```

Reply:
Got it. thank you!